contrib/storage-http/README.md

```diff
@@ -45,6 +45,30 @@ The `connection` property can accept the following options.
 
 `url`: The base URL which Drill will query.
 
+##### Parameters in the URL
+Many APIs require parameters to be passed directly in the URL instead of as query arguments.  For example, github's API allows you to query an organization's repositories with the following
+URL:  https://github.com/orgs/{org}/repos
+
+As of Drill 1.20.0, you can simply set the URL in the connection using the curly braces.  If your API includes URL parameters you MUST
+include them in the `WHERE` clause in your query, or Drill will throw an error. 
```

Comment:
Here we say "MUST", but below we walk it back. Can we improve the flow?

Reply:
Fixed